tests/test_reduction_ops.py

```diff
@@ -353,6 +353,37 @@ def test_accuracy_layernorm(shape, dtype):
     gems_assert_close(res_bias_grad, ref_bias_grad, dtype, reduce_dim=M)
 
 
+@pytest.mark.parametrize("shape", REDUCTION_SHAPES)
```

Comment:
Please add @pytest.mark.weight_norm_interface here, and move this function to test_norm_ops.py
Please rebase FlagOpen:master, and we will merge this pr soon.

Reply:
done